Resources/Prototypes/_Sunrise/Entities/Clothing/Eyes/glasses.yml

```
@@ -74,15 +74,36 @@
     - WhitelistChameleon
 
 - type: entity
-  parent: ClothingEyesGlassesCheapSunglasses
+  parent: [BaseNightVisionDevice, ClothingEyesGlassesCheapSunglasses, PowerCellSlotSmallItem]
   id: ClothingEyesGlassesNVG
   name: sun glasses
   description: A pair of black sunglasses.
   components:
   - type: ShowSyndicateIcons
```

YAMLlint (1.37.1)

[error] wrong indentation: expected 4 but found 2

(indentation)